src/pages/dmt5/_margin_calculator.tsx

```diff
@@ -9,6 +9,18 @@ import { Localize, localize } from 'components/localization'
 import device from 'themes/device'
 import { DerivStore } from 'store'
 
+interface CalculatorProps {
+    children?: React.ReactNode
+    key?: number
+    index?: number
+    name: ReactElement
+    text: ReactElement
```

Comment:
Please keep it consistent either by destructuring out `ReactElement` & `ReactNode` or calling `React.` on it.

Reply:
Done @sean-binary. Decided to use React. to not to import methods every time